WickedEngine/wiArchive.h

```diff
@@ -5,6 +5,10 @@
 #include <string>
 #include <vector>
 
+#ifndef _WIN32
+#include <cstring>
+#endif
+
```

Comment:
We could remove the `#ifdef` here, even if the Windows string/vector already includes `<string>` beforehand?

Reply:
We'll have to try that after I get the initial pull request accepted.